website/blog/2022-01-19-version-067.md

```diff
@@ -0,0 +1,78 @@
+- [Interested in helping React Native stabilise new releases?](/blog/2022/01/19/version-067#interested-in-helping-react-native-stabilise-new-releases)
+
+<!--truncate-->
+
+## Highlights of 0.67
+
+As mentioned in Meta's [H2 2021 plans](https://reactnative.dev/blog/2021/08/19/h2-2021), React Native is attempting more frequent releases for a shorter turnaround time for new features and fixes (like the new architecture) to land in the community. Naturally, many releases will focus on fixes and improvements.
+
+Here are some notable changes coming in 0.67.0:
+
+- Lean-core removals: [DatePickerAndroid](https://github.com/facebook/react-native/commit/7a770526c626e6659a12939f8c61057a688aa623#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346)
+- Rename deprecated Keyboard.removeEventListener to Keyboard.removeListener. [RN: Rename `Keyboard.remove{Event =>}Listener`](https://github.com/facebook/react-native/commit/4ad177efc7ad6a39b37103978c68c9b4d2b09c98)
+- Bump Gradle version to 7.2, Bump Kotlin version to 1.5.31 [Bump Kotlin and Gradle versions (#32319)](https://github.com/facebook/react-native/commit/9ae3367431428748f5486c782199beb4f9c6b477)
+- A notable callout: 0.67 continues to depend on Hermes 0.9.0 (unchanged from 0.66) - there was a [regression for iOS usage of Hermes 0.10](https://github.com/facebook/hermes/issues/649) – which will be fixed in Hermes 0.11 which will be included in the 0.68 release of React Native.
+
+You can find the [full changelog here](https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0670).
+
+You can participate in the conversation on the status of this release at [this discussion](https://github.com/reactwg/react-native-releases/discussions/10) - and, as always, to help you upgrade to this version, you can use the [upgrade helper](https://react-native-community.github.io/upgrade-helper/) ⚛️
+
+### Acknowledgements
+
+This release includes [379 commits with 74 contributors](https://github.com/facebook/react-native/compare/0.66-stable...0.67-stable)! Thank you, to all our contributors (old and new)! You can find the [full changelog here](https://github.com/facebook/react-native/blob/main/CHANGELOG.md#v0670).
+
+We wanted to also thank the release testers who helped us make sure that 0.67.0 could reach your codebases without any massive regression. Specifically, we wanted to thank:
+
+- Marc Rousavy ([@mrousavy](https://github.com/mrousavy)) from [Margelo](https://margelo.io/), that surfaced the Hermes 0.10 problem (that would have never been caught on CI testing)
+- The Reanimated team for quickly preparing a [0.67 compatible version](https://github.com/software-mansion/react-native-reanimated/releases/tag/2.2.4) of their lib early in the 0.67 RC phase.
+- Elias Nahum ([@enahum](https://github.com/enahum)) from [Mattermost](https://mattermost.com/)
+- Mike Hardy ([@mikeHardy](https://github.com/mikeHardy)) working with [Invertase](https://invertase.io/)
```

Comment:
Let's get the names of all our release testers here -- if you have a list, I can add them in

Reply:
```suggestion
- Mike Hardy ([@mikeHardy](https://github.com/mikeHardy)) working with [Invertase](https://invertase.io/)

We appreciate also [Rainbow](https://rainbow.me/), [Comm](https://comm.app/) and [Ledger Live](https://www.ledger.com/ledger-live) for also being part of the pilot of the "Release Tester" program (more details below).
```
as agreed 👆